README.md

```diff
@@ -1,90 +1,72 @@
+Modern applications are build on existing systems such as S3, EC2, and Cloud Foundry. Documentation for how these underlying systems fulfill NIST controls or PCI SSC Data Security Standards is a prerequisite for receiving authorization to operate (ATO).
 
-Control Masonry simplifies the process of creating control documentations by providing:
-1. a control justification store for each of multiple systems in individual yaml files
-2. a system for government project to edit existing files and also add new control files for their applications and organizations.
-3. a script for combining the files into a single base file
-4. a pipeline for generating clean and standardized documentation
+Control Masonry simplifies the process of certification documentations by providing:
+1. a data store for certifications (ex FISMA), standards (ex NIST-800-53), and the individual system components (ex AWS-EC2).
+2. a way for government project to edit existing files and also add new control files for their applications and organizations.
+3. a pipeline for generating clean and standardized certification documentation
 
-## Installation & Usage
-Control Masonry runs on Node.js
+## Components Documentation
+Component documentation contains information about individual system components and the standards they satisfy.
 
-##### To install requirements
-```bash
-npm install
+```yaml
+name: User Account and Authentication (UAA) Server
+references:
+- reference_name: User Account and Authentication (UAA) Server
+  reference_url: http://docs.pivotal.io/pivotalcf/concepts/architecture/uaa.html
+- reference_name: Creating and Managing Users with the UAA CLI (UAAC)
+  reference_url: http://docs.pivotal.io/pivotalcf/adminguide/uaa-user-management.html
+governors:
+- governor_name: Cloud Foundry Roles
+  governor_url: https://cf-p1-docs-prod.cfapps.io/pivotalcf/concepts/roles.html
+- governor_name: Cloud Foundry Org Access
+  governor_url: https://github.com/cloudfoundry/cloud_controller_ng/blob/master/spec/unit/access/organization_access_spec.rb
+- governor_name: Cloud Foundry Space Access
+  governor_url: https://github.com/cloudfoundry/cloud_controller_ng/blob/master/spec/unit/access/space_access_spec.rb
+satisfies:
+  AC-2: Cloud Foundry accounts are managed through the User Account and Authentication
+    (UAA) Server.
+  IA-2: The UAA is the identity management service for Cloud Foundry. Its primary
+    role is as an OAuth2 provider, issuing tokens for client applications to use when
+    they act on behalf of Cloud Foundry users.
+  SC-13: All traffic from the public internet to the Cloud Controller and UAA happens
+    over HTTPS and operators configure encryption of the identity store in the UAA
+  SC-28 (1): Operators configure encryption of the identity store in the UAA. When
+    users register an account with the Cloud Foundry platform, the UAA, acts as the
```

Comment:
I really think we need a hierarchy of "components" - UAA is a component of CF, but most government CF installations actually use the UAA plugin mechanism to connect external IDP systems such as CA Siteminder (because they fulfill all the 800-53 AC controls that way). So I think the "component" that satisfies AC-2 is a sub-component of UAA.